engine/consensus/compliance/core.go

```diff
@@ -413,14 +418,14 @@ func checkForAndLogOutdatedInputError(err error, log zerolog.Logger) bool {
 	return false
 }
 
-// checkForAndLogInvalidInputError checks whether error is an `engine.InvalidInputError`.
+// checkForAndLogUnverifiableInputError checks whether error is an `engine.UnverifiableInputError`.
 // If this is the case, we emit a log message and return true.
-// For any error other than `engine.InvalidInputError`, this function is a no-op
+// For any error other than `engine.UnverifiableInputError`, this function is a no-op
 // and returns false.
-func checkForAndLogInvalidInputError(err error, log zerolog.Logger) bool {
-	if engine.IsInvalidInputError(err) {
-		// the block is invalid; log as error as we desire honest participation
-		log.Err(err).Msg("received invalid block from other node (potential slashing evidence?)")
+func checkForAndLogUnverifiableInputError(err error, log zerolog.Logger) bool {
+	if engine.IsUnverifiableInputError(err) {
+		// the block cannot be validated
+		log.Err(err).Msg("received unverifiable block proposal; this is an indicator of a proposal that cannot be verified under current state")
```

Comment:
* I feel adding an interpretation what might be the cause would be very helpful
* This could happen if the node is behind following main consensus. Hence, I would suggest to log with `Warn` level 
```suggestion
		log.Err(err).Msg("received unverifiable block proposal; this might be an indicator that a malicious proposer is generating detached blocks very far ahead")
```